lib/main.dart

```diff
@@ -22,27 +24,36 @@ class MyApp extends StatelessWidget {
         builder: (context, state) => const InitialScreen(),
       ),
       GoRoute(
-        path: '${screenRoutes[Screen.initial]}${screenRoutes[Screen.welcome]}', 
+        path: '/${screenRoutes[Screen.welcome]}', 
         builder: (context, state) => const WelcomeScreen(),
       ),
       GoRoute(
-        path: '${screenRoutes[Screen.initial]}${screenRoutes[Screen.login]}',  
+        path: '/${screenRoutes[Screen.login]}',  
         builder: (context, state) => const LoginScreen(),
       ),
       GoRoute(
-        path: '${screenRoutes[Screen.initial]}${screenRoutes[Screen.register]}',
+        path: '/${screenRoutes[Screen.register]}',
         builder: (context, state) => const RegisterScreen(),
       ),
-      GoRoute(
-        path: '${screenRoutes[Screen.initial]}${screenRoutes[Screen.home]}',
-        builder: (context, state) => const HomeScreen(),
+      ShellRoute(
+        builder: (context, state, child) {
+          return HomeScreen(child: child);
+        },
         routes: [
           GoRoute(
-            path: screenRoutes[Screen.editSettings]!,
-            builder: (context, state) => const EditSettingsScreen(),
+            path: '/${screenRoutes[Screen.home]}', 
+            builder: (context, state) => const HomeTab(),
```

Comment:
It seems like you have forgotten to add the file to git

Reply:
added.
thanks